packages/front-end/components/Experiment/TabbedPage/index.tsx

```diff
@@ -323,6 +324,9 @@ export default function TabbedPage({
             safeToEdit={safeToEdit}
           />
         </div>
+        <div className={tab === "health" ? "d-block" : "d-none d-print-block"}>
```

Comment:
Nit: It looks like we need `mt-3` here. When displaying the error or alert divs, it's overlapping the tabs just a bit.

E.G.
`<div className={tab === "health" ? "d-block mt-3" : "d-none d-print-block"}>`

There may be a better place to add the top margin, just wanted to call it out.

Reply:
<img width="546" alt="Screen Shot 2023-11-15 at 1 39 03 PM" src="https://github.com/growthbook/growthbook/assets/75274610/8bda1551-dc4e-43bd-bcb4-8b701d591171">
<img width="426" alt="Screen Shot 2023-11-15 at 1 38 50 PM" src="https://github.com/growthbook/growthbook/assets/75274610/3ef46aea-ce2e-4d71-be27-2d20398e24c0">